webtop/init/provision_docker_groups.sh

```diff
@@ -6,17 +6,16 @@ log() {
   fi
 }
 
-log "Provisioning docker groups"
-sudo groupadd -g 999 docker-ubuntu
-sudo groupadd -g 992 docker-lsio
-sudo groupadd -g 130 docker-plex
+# Retrieve the correct GID from the mounted docker.sock
+DOCKER_GROUP_GID=$(stat -c '%g' /var/run/docker.sock)
```

Comment:
Clever! I spent a bit of time trying to solve it from a very wrong perspective of augmenting permissions on the socket file itself, but ended up not doing that out of fears to mess up something on the User's host. This way is much cleaner ✨